fortinet_fortigate/plugin.spec.yaml

```
@@ -739,21 +738,22 @@ actions:
         description: Information about the operation that was performed
         type: object
         required: true
-
   add_address_object_to_address_group:
     title: Add Address Object to Group
     description: Add an address object to an address group
     input:
-      group_name:
-        title: Group Name
+      group:
+        title: Group
         description: Group name
         type: string
         required: true
-      address_object_name:
-        title: Address Object Name
-        description: Address object name
+        example: blocked IP's
```

Has a better ring to it :)

```suggestion
        example: InsightConnect Block List
```